ui/js/graphs.js

```diff
@@ -688,6 +688,9 @@ perf.controller('GraphsCtrl', [
                 timeRange: function() {
                   return $scope.myTimerange.value;
                 },
+                testCounter: function() {
```

Comment:
yep, I just try it with just set the value of $scope.seriesList.length, but it tells me the equation can't be recognized. But I had changed the name of testCounter to numTestsDisplayed :)